src/object-marker.scss

```diff
@@ -0,0 +1,85 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-marker
+*/
+$block: #{$fd-namespace}-object-marker;
+
+.#{$block} {
+  // ICON VARIABLES
+  $fd-object-marker-font-size: 1rem;
+  $fd-object-marker-color: var(--sapContent_MarkerIconColor);
+  $fd-object-marker-text-color: var(--sapContent_LabelColor) !default;
+  $fd-object-marker-icon-padding: 0.25rem !default;
+  $fd-object-marker-padding-right: 0.25rem !default;
+  // TEXT VARIABLES
+  $fd-object-marker-font-family: var(--sapFontFamily) !default;
+
+  @include fd-reset();
+
+  font-family: $fd-object-marker-font-family;
+  font-size: $fd-object-marker-font-size;
+  max-width: 100%;
+  color: $fd-object-marker-text-color;
+  word-break: break-word;
+  display: inline-flex;
+  align-items: center;
+  line-height: 1;
+
+  &::before {
+    color: $fd-object-marker-color;
+    padding-right: $fd-object-marker-padding-right;
+  }
+
+  // CLICKABLE OBJECT STATUS
+  &--link {
+    display: inline-block;
+    text-decoration: none;
+    color: $fd-object-marker-color;
+    cursor: pointer;
+
+    &::before {
+      position: relative;
+      top: 0.125rem;
+    }
+
+    &:hover {
+      text-decoration: underline;
+
+      &::before {
+        text-decoration: none;
+      }
+    }
+
+    &:visited {
+      text-decoration: none;
+    }
+  }
+
+  // ICON ONLY MODE
+  @include fd-empty() {
+    justify-content: center;
+    padding-right: 0;
+    padding-left: 0;
+
+    &::before {
+      padding-right: 0;
+      padding-left: 0;
```

Comment:
<img width="406" alt="Screen Shot 2020-07-02 at 2 48 46 PM" src="https://user-images.githubusercontent.com/39598672/86398160-29fe1600-bc73-11ea-99ef-07c74e54abfa.png">
 this part is missing